src/main/java/seedu/storemando/logic/parser/SortCommandParser.java

```diff
@@ -21,9 +22,13 @@ public SortCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
         if (trimmedArgs.equalsIgnoreCase(EXPIRYDATE_KEYWORD)) {
             return new SortExpiryDateCommand();
-        } else if (trimmedArgs.equalsIgnoreCase(QUANTITY_ASCENDING_KEYWORD)) {
+        }
+        String[] splitWords = trimmedArgs.split("\\s+");
+        if (splitWords.length == 2 && splitWords[0].equalsIgnoreCase(QUANTITY_KEYWORD)
+            && splitWords[1].equalsIgnoreCase(ASCENDING_KEYWORD)) {
             return new SortQuantityCommand(true);
```

Comment:
Prevent using magic boolean. Other than that LGTM

Reply:
ok will make the changes